.github/workflows/cargo_machete.yml

```diff
@@ -0,0 +1,17 @@
+name: Cargo Machete
```

Comment:
it's not part of the reusable checks file because you want to run it _only_ on PRs?
Hm I guess another advantage is that this way we don't have to care about contributor vs maintainer ci.

Reply:
I wanted to copy-paste it from another repository without wading into the depths of our CI 😬